pkgs/development/libraries/physics/cernlib/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, gfortran, gnumake, imake, makedepend, motif, xorg }:
+
+stdenv.mkDerivation rec {
+  version = "2006";
+  name = "cernlib-${version}";
+
+  src = fetchurl {
+    url = "http://cernlib.web.cern.ch/cernlib/download/${version}_source/tar/${version}_src.tar.gz";
+    sha256 = "0awla1rl96z82br7slcmg8ks1d2a7slk6dj79ywb871j2ksi3fky";
+  };
+
+  buildInputs = with xorg; [ gfortran gnumake imake makedepend motif libX11 libXft libXt ];
```

Comment:
If it's a compiler you want it to be the native one. Eventually the libs will hopefully be separated out, but for now that should work (it certainly will work with non-cross).

GNU Make should be a build-time dep because it runs on the build platform (host -> build, so -1), gfortran likewise because that and because it targets our host platform (target -> host, also -1). Normal run-time deps run on the run are host platform (host -> host, so 0).